site/content/pages/organizing/04-Running your event/07-video-streaming.txt

```diff
@@ -0,0 +1,17 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Video - recording and streaming
+---
+
+Whether or not you're able to livestream, it's important to record all the talks. This is invaluable for your speakers, and it's great for the community. Your audiovisual company should be able to provide a camera or cameras and record the feed off the board into a computer.
```

Comment:
If it's required, then we should put it in the rules. That does raise the barrier for entry a bit.